### PR TITLE
update NL translation for Supportpal 3.6

### PR DESCRIPTION
**Pull request by TimVNL**

### CODE REVIEW

**Comment by bytestream** [comment]
Thanks @TimVNL :raised_hands:

File: core.php [hunk 1]
```diff
@@ -504,23 +504,23 @@
...
      */
-    "results_per_page"          => "Results Per Page",
-    "class_name"                => "Class Name",
-    "possible_permission_issue" => "You are running the command as \":cli_user\", while \":path\" is owned by \":path_user\", this can lead to file permission issues.",
-    "system_update"             => "System Update",
-    "system_update_desc"        => "Update SupportPal to the latest version available.",
-    "update_available"          => "Update Available",
+    "results_per_page"          => "Resultaten per pagina",
+    "class_name"                => "Klasse Naam",
+    "possible_permission_issue" => "U voert de opdracht uit als  \":cli_user\", terwijl \":path\" is eigendom van \":path_user\", dit kan leiden tot problemen met bestandsrechten.",
```
Comment (ju5t) [inline core.php:509]: @TimVNL correct me if I'm wrong but I think it should be:

```suggestion
    "possible_permission_issue" => "U voert de opdracht uit als \":cli_user\", terwijl \":path\" eigendom is van \":path_user\", dit kan leiden tot problemen met bestandsrechten.",
```

File: core.php [hunk 2]
```diff
@@ -504,23 +504,23 @@
...
-    "cron_must_run_for_update"  => "The cron job must be running to use this function.",
-    "support_expired"           => "The support and updates on your license have expired.",
-    "update_permitted_version"  => "The license only permits updating to :version.",
-    "update_renew_support"      => "To update to the latest version of SupportPal, please renew your support and updates.",
-    "renew"                     => "Renew",
+    "cron_must_run_for_update"  => "De Cron Job moet actief zijn om deze functie te gebruiken .",
+    "support_expired"           => "De ondersteuning en updates van uw licentie zijn verlopen.",
+    "update_permitted_version"  => "De licentie staat alleen updaten toe naar :version.",
+    "update_renew_support"      => "Vernieuw uw ondersteuning en updates om bij te werken naar de nieuwste versie van SupportPal.",
+    "renew"                     => "Vernieuw",
```
Comment (ju5t) [inline core.php:524]: Don't want to nitpick but I think `Verleng` instead of `Vernieuw` has a better ring to it.

File: installer.php [hunk 3]
```diff
@@ -166,26 +166,26 @@
...
-    "current_and_new_version"   => "You have version :current_version. Latest version available is :new_version.",
-    "update_to_version"         => "You have version :current_version. Would you like to update to :new_version?",
-    "downloading_version"       => "Downloading version :version to :path...",
-    "checksum_not_matched"      => "The downloaded file failed checksum verification.",
-
-    "check_readiness"           => "Checking if the application is able to upgrade...",
-    "read_only"                 => "The filesystem at \":path\" is read only. Updates cannot be applied against a read only file system.",
-    "file_ownership"            => "File \":path\" is owned by \":owner\" but PHP is running as \":actual\". The file owner and PHP user must be the same during an update for all application files.",
-    "minimum_disk_space"        => "Directory \":path\" has insufficient space. There must be at least :space megabytes free.",
+    "suggest_run_update"        => "Commando niet beschikbaar. Voer alsjeblieft app:update eom te downloaden en te upgraden naar de nieuwste beschikbare versie : %s.",
```
Comment (ju5t) [inline installer.php:169]: This feels a bit too literal.

```suggestion
    "suggest_run_update" => "Commando niet beschikbaar. Voer alsjeblieft app:update om te upgraden naar de nieuwste beschikbare versie: %s.",
```

File: installer.php [hunk 4]
```diff
@@ -166,26 +166,26 @@
...
+    "suggest_run_upgrade"       => "Er wordt al een upgrade uitgevoerd. Voer alsjeblieft app:upgrade.",
```
Comment (ju5t) [inline installer.php:170]:
```suggestion
    "suggest_run_upgrade" => "Er wordt al een upgrade uitgevoerd. Voer alsjeblieft app:upgrade uit.",
```

Not 100% sure about this sentence.

File: installer.php [hunk 5]
```diff
@@ -166,26 +166,26 @@
...
+    "failed_to_download"        => "Kan bestand niet downloaden :message",
+    "backup_failed_abort"       => "Back-up mislukt. Update wordt afbreken.",
```
Comment (ju5t) [inline installer.php:172]:
```suggestion
    "backup_failed_abort" => "Back-up mislukt. Update wordt afgebroken.",
```

File: installer.php [hunk 6]
```diff
@@ -166,26 +166,26 @@
...
+    "update_failed"             => "Systeemupdate mislukt: :message.",
+    "command_failed"            => "Commando :command, mislukt met output: :message.",
+    "already_latest_version"    => "U gebruikt al de nieuwste versie (:version).",
```
Comment (ju5t) [inline installer.php:175]: I think 'je' makes more sense as everything else is translated to `je` as well. I tend to make this mistake a lot, too :)

File: installer.php [hunk 7]
```diff
@@ -166,26 +166,26 @@
...
+    "update_not_supported"      => "Automatische update naar de nieuwste versie (:version) wordt niet ondersteund.",
+    "current_and_new_version"   => "Je hebt versie :current_version. De nieuwste beschikbare versie is  :new_version.",
+    "update_to_version"         => "Je hebt versie :current_version. Wil je updaten naar :new_version?",
+    "downloading_version"       => "Versie :version wordt gedownload naar :path...",
+    "checksum_not_matched"      => "Het gedownloade bestand is mislukt voor controlesomverificatie .",
```
Comment (ju5t) [inline installer.php:180]: controlesom verificatie, it should have a space. To be honest I think this is too technical both in English and in Dutch.

**Comment by bytestream** [comment]
Submit a PR @ju5t and I'll give @TimVNL time to :+1: or :-1: :)

File: installer.php [hunk 8]
```diff
@@ -166,26 +166,26 @@
...
+
+    "check_readiness"           => "Controleren of de toepassing kan upgraden... ",
```
Comment (ju5t) [inline installer.php:182]: Both `application` and `toepassing` are bad choices in my opinion. But if you want to translate it, `applicatie` probably sounds better.

**Comment by ju5t** [comment]
@bytestream I think for translation PR's it always makes sense to have someone review it. I think this was merged too soon :)

**Comment by bytestream** [comment]
> @bytestream I think for translation PR's it always makes sense to have someone review it. I think this was merged too soon :)

I agree  wholeheartedly however 100% of the time no one comments, and this isn't Tim's first rodeo - https://github.com/supportpal/language-files/pulls?q=is%3Apr+is%3Aclosed+author%3Atimvnl. You just bucked the trend in this case.

If you're happy to review PRs to the `nl` branch then I'll mention from you in the future.

**Comment by TimVNL** [comment]
I will take a look at the suggestions and make a New pull request tommorow a fresh pair of eyes is always welcome :) 

In this case the pull request was maybe a bit rushed my appologies

File: installer.php (hunk 6 above)
Comment (TimVNL) [reply installer.php:175]: Agreed

**Comment by ju5t** [comment]
I can help too. I don't always have time, but if you get to it earlier @TimVNL feel free to ping me. Happy to help.

**Pull request merged**